lib/unhangout-routes.js

```diff
@@ -788,6 +795,136 @@ module.exports = {
         // Starting and stopping events
         //
 
+        app.get("/followup/event/:id/participant_:participantCount", function(req, res) {
+
+            var event = db.events.get(req.params.id);
+
+            var eventTitle = event.get("title");
+            var history = event.get("history");
+            var participantCount = req.params.participantCount; 
+
+            var allUserIds = [];
+            allUserIds.push(null);
+
+            _.each(history.event, function(elapsedTime, userId) {
+                
+                //Get the user object for a specific ID 
+                var user = db.users.get(userId);
+
+                // Find all userIDs for users who shared sessions with me at this event.
+                var userIds = event.getUserIdsSharingSessionsWith(userId);
+                
+                // get user objects for the IDs.
+                var users = _.map(userIds, function(userId) { 
+                    return db.users.get(userId); 
+                });
+
+                allUserIds.push({ user: user, users: users });
```

Comment:
It looks to me as though the preview view is only using a single entry from ``allUserIds`` (specifically ``allUserIds[participantCount]`` at once.  If this is true, there is no need to build a data structure with every user in it just to display data from the one user.